### ISSUE TITLE
dunst not working with example config (Warning: Unknown keyboard shortcut: ctrl+grave)

### ISSUE THREAD

**Issue by jacksgt** [dunst-project/dunst#268]
Hi,

I recently set up dunst. While configuring it I noticed it wasn't working wit the default (supplied) config.
Specifically, the issue was the following line:

```
# Redisplay last message(s).
# On the US keyboard layout "grave" is normally above TAB and left
# of "1".
history = ctrl+grave
```

I got the following error message:

```
$ dunst -config dunstrc.example
Warning: Unknown keyboard shortcut: ctrl+grave
```

The "problem" is I am using a default german keyboard layout, which does not have a 'grave'-key:

```
$ xmodmap -pke | grep grave
keycode  21 = dead_acute dead_grave dead_acute dead_grave dead_cedilla dead_ogonek dead_cedilla
```

Thanks to @stapelberg for figuring this out: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=829385

I think we should document this behavior in the config file itself. While the issue is simple and logical, its cause is probably not immediately clear to most users.

**TL;DR**:
Add a short comment to the default config:

```
# Redisplay last message(s).
# On the US keyboard layout "grave" is normally above TAB and left
# of "1". Make sure this key actually exists on your keyboard layout,
# e.g. check the output of 'xmodmap -pke'
history = ctrl+grave
```
